packages/react/src/components/FeatureFlags/index.js

```diff
@@ -65,9 +85,19 @@ FeatureFlags.propTypes = {
   children: PropTypes.node,
 
   /**
-   * Provide the feature flags to enabled or disabled in the current React tree
+   * Provide the feature flags to enabled or disabled in the current Rea,ct tree
    */
-  flags: PropTypes.objectOf(PropTypes.bool),
+  flags: deprecate(
+    PropTypes.objectOf(PropTypes.bool),
+    'The `flags` prop for `FeatureFlag` has ' +
+      'been deprecated. Please run the `featureflag-deprecate-flags-prop` codemod to migrate to individual boolean props.'
```

Comment:
Once added to the packages/upgrade/src/upgrades.js, update this to have the full command they can copy and paste to run

```
npx @carbon/upgrade migrate featureflag-deprecate-flags-prop --write
```